src/server/editorServices.ts

```diff
@@ -1006,14 +1006,15 @@ namespace ts.server {
 
         /**
          * @param filename is absolute pathname
+         * @param fileContent is a known version of the file content that is more up to date

```

Comment:
Is "more up to date than the one on disk" making sense?
So the situation is that the server may have crashed before the client saves the edited buffer in sublime or vs code, therefore when restarting the server, the buffer content should take higher priority.